### PR TITLE
[FLINK-30007] Explain how users can request a Jira account

### PR DESCRIPTION
**Pull request by MartijnVisser**
* ~Looking for a Chinese translation (this was just provided by Google Translate)~
* ~Pending implementation of https://issues.apache.org/jira/browse/INFRA-23898~

### CODE REVIEW

File: community.zh.md [hunk 1]
```diff
@@ -167,6 +167,49 @@ Committer 们会关注 [Stack Overflow](http://stackoverflow.com/questions/tagge
...
+```text
+Subject: Add me as a contributor to JIRA
+
+Hello,
+
+Please add me as a contributor to JIRA.
+My JIRA username is: [INSERT YOUR JIRA USERNAME HERE]
+
+Thanks,
+[INSERT YOUR NAME HERE]
```
Comment (leonardBang) [inline community.zh.md]:
```suggestion
所有 issue 必须使用英文。

您需要拥有 JIRA 账号才能记录案例和 issue。

### 我已经有了一个 ASF JIRA 账号，希望将账号添加为贡献者

如果您已经拥有 ASF JIRA 账号，则无需注册新账号。
请按照下述邮件模板向 [jira-requests@flink.apache.org]([jira-requests@flink.apache.org]) 地址发送电子邮件，以便我们将您的账号添加到 JIRA 中的贡献者列表：

[在您的电子邮件客户端中打开模板](mailto:jira-requests@flink.apache.org?subject=Add%20me%20as%20a%20contributor%20to%20JIRA&amp;body=Hello,%0A%0APlease%20add%20me%20as%20a%20contributor%20to%20JIRA.%0AMy%20JIRA%20username%20is:%20%5BINSERT%20YOUR%20JIRA%20USERNAME%20HERE%5D%0A%0AThanks,%0A%5BINSERT%20YOUR%20NAME%20HERE%5D)

```text
主题: Add me as a contributor to JIRA

Hello,

Please add me as a contributor to JIRA.
My JIRA username is: [你的 JIRA 用户名]

Thanks,
[你的名字]
```
```

File: community.zh.md [hunk 2]
```diff
@@ -167,6 +167,49 @@ Committer 们会关注 [Stack Overflow](http://stackoverflow.com/questions/tagge
...
+
+Hello,
+
+I would like to request a JIRA account.
+My proposed JIRA username: [INSERT YOUR DESIRED JIRA USERNAME HERE (LOWERCASE LETTERS AND NUMBERS ONLY)]
+My full name: [INSERT YOUR FULL NAME HERE]
+My email address: [INSERT YOUR EMAIL ADDRESS HERE]
+
+Thanks,
+[INSERT YOUR NAME HERE]
```
Comment (leonardBang) [inline community.zh.md]:
```suggestion
### 我没有 ASF JIRA 账号，想申请一个账号并将其添加为贡献者

为了申请 ASF JIRA 账号，您需要按照下述邮件模板向 [jira-requests@flink.apache.org]([jira-requests@flink.apache.org]) 发送电子邮件：

[在您的电子邮件客户端中打开模板](mailto:jira-requests@flink.apache.org?subject=Request%20for%20JIRA%20Account&amp;body=Hello,%0A%0AI%20would%20like%20to%20request%20a%20JIRA%20account.%0AMy%20proposed%20JIRA%20username:%20%5BINSERT%20YOUR%20DESIRED%20JIRA%20USERNAME%20HERE%20(LOWERCASE%20LETTERS%20AND%20NUMBERS%20ONLY)%5D%0AMy%20full%20name:%20%5BINSERT%20YOUR%20FULL%20NAME%20HERE%5D%0AMy%20email%20address:%20%5BINSERT%20YOUR%20EMAIL%20ADDRESS%20HERE%5D%0A%0AThanks,%0A%5BINSERT%20YOUR%20NAME%20HERE%5D)

```text
主题: Request for JIRA Account

Hello,

I would like to request a JIRA account.
My proposed JIRA username: [ 填写你期望的 JIRA 用户名 (只能包含小写字母和数字)]
My full name: [填写你的全名]
My email address: [填写你的邮件地址]

Thanks,
[你的名字]
```

**Review by knaufk — APPROVED** [review]
LGTM

File: community.md [hunk 3]
```diff
@@ -170,6 +170,47 @@ Make sure to tag your questions there accordingly to get answers from the Flink
...
+
+[Open the template in your email client](mailto:jira-requests@flink.apache.org?subject=Request%20for%20JIRA%20Account&amp;body=Hello,%0A%0AI%20would%20like%20to%20request%20a%20JIRA%20account.%0AMy%20proposed%20JIRA%20username:%20%5BINSERT%20YOUR%20DESIRED%20JIRA%20USERNAME%20HERE%20(LOWERCASE%20LETTERS%20AND%20NUMBERS%20ONLY)%5D%0AMy%20full%20name:%20%5BINSERT%20YOUR%20FULL%20NAME%20HERE%5D%0AMy%20email%20address:%20%5BINSERT%20YOUR%20EMAIL%20ADDRESS%20HERE%5D%0A%0AThanks,%0A%5BINSERT%20YOUR%20NAME%20HERE%5D)
+
+```text
+Subject: Request for JIRA Account
+
+Hello,
+
+I would like to request a JIRA account.
+My proposed JIRA username: [INSERT YOUR DESIRED JIRA USERNAME HERE (LOWERCASE LETTERS AND NUMBERS ONLY)]
```
Comment (leonardBang) [inline community.md:207]: User need try this process again if username conflicts happens? 
Could we offer guidance to check the desired username will be conflicted with existed usernames or not?

**Review by leonardBang — COMMENTED** [review]
Thanks @MartijnVisser for the contribution, I improved the Chinese doc a little, and I also left comments about the username conflicts concern.

File: community.md (hunk 3 above)
Comment (MartijnVisser) [reply community.md:207]: Let's see if we run into a lot of conflicts first. If we do, then let's change the docs with a little howto check if the username is already taken

Comment (leonardBang) [reply community.md:207]: Okay, it makes sense to me

**Review by leonardBang — APPROVED** [review]
Thanks @MartijnVisser for the update, LGTM.

**Comment by zentol** [comment]
Might want to add a note that users who only wish to report a bug may do so via the user mailing list/slack instead of jumping through these hoops.

**Pull request merged**